Create ui-c8y-1018-503-105-adding-no-conflict-to-avoid-coliding-leaflet-versions-6622-graft-release.md (#2271)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1018-503-105-adding-no-conflict-to-avoid-coliding-leaflet-versions-6622-graft-release.md

---
date: ""

> title: adding noConflict to avoid coliding leaflet versions (#6622) [GRAFT][release/y2024] (#6650)

Author: The Claude AI suggested to use the following title:
```suggestion
title: Avoid colliding Leaflet versions by adding noConflict
```

product_area: Application enablement & solutions
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-YbYJ3gLU_
    label: Web SDK
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-59809
version: 1018.503.105
---

> adding noConflict to avoid coliding leaflet versions (#6622) [GRAFT][release/y2024] (#6650)

Author: The Claude AI suggested to use the following body:
```suggestion
In some scenarios, different versions of the Leaflet library used by Cumulocity IoT and custom widgets could collide and lead to unexpected behavior or errors. To address this, the noConflict mode has been added to Cumulocity IoT's Leaflet integration. With this change, custom widgets can now use their own Leaflet version without interfering with Cumulocity IoT's Leaflet version. This improves the reliability and compatibility when using Leaflet in custom widgets alongside Cumulocity IoT's built-in geospatial features.
```

> In some scenarios, different versions of the Leaflet library used by Cumulocity IoT and custom widgets could collide and lead to unexpected behavior or errors. To address this, the noConflict mode has been added to Cumulocity IoT's Leaflet integration. With this change, custom widgets can now use their own Leaflet version without interfering with Cumulocity IoT's Leaflet version. This improves the reliability and compatibility when using Leaflet in custom widgets alongside Cumulocity IoT's built-in geospatial features.

Reviewer: ```suggestion
In some scenarios, different versions of the Leaflet library used by {{< product-c8y-iot >}} and custom widgets could collide and lead to unexpected behavior or errors. To address this, the noConflict mode has been added to {{< product-c8y-iot >}}'s Leaflet integration. With this change, custom widgets can now use their own Leaflet version without interfering with {{< product-c8y-iot >}}'s Leaflet version. This improves the reliability and compatibility when using Leaflet in custom widgets alongside {{< product-c8y-iot >}}'s built-in geospatial features.
```
